dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/DataOwnerControllerTest.kt

```diff
@@ -134,11 +145,13 @@ class DataOwnerControllerTest {
     fun `post get head and delete data owners to an unknown company`() {
         jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(TechnicalUser.Admin)
         val randomCompanyId = UUID.randomUUID()
-        val userId = UUID.randomUUID()
+        val userId = UUID.fromString(
```

Comment:
Given the new functionality of sending an e-mail for data ownership acceptance, a random userID throws a ServerException when calling the method postDataOwner. This is because the random userID actually does not exist, which leads to a failure when attempting to send the e-mail. Therefore, an existing userID is required. The e2e tests were adjusted accordingly.